docs/decision_record.md

```diff
@@ -6,6 +6,97 @@
 
 [TOC]
 
+## 2022-09-05 - How sequence collection are serialized prior to digestion
+
+### Decision
+
+The serialisation of a sequence collection will use the following steps
+
+ 1. Apply RFC-8785 on each array of level 2
+ 2. Digest the canonical representation of each array
+ 3. Create object representation of the seq-col using array names and digested arrays
+ 4. Apply RFC-8785 on the object representation
+ 5. Digest the final canonical representation
+
+
+#### 1. Apply RFC-8785 for converting from level 2 to level 1
+
+For example the length array at level 2:
+```json
+[248956422, 242193529, 198295559]
+```
+
+Will be serialised using RFC-8785 and digested as a binary string. Here the output of the python implementation: 
+
+```python
+b'[248956422,242193529,198295559]'
+```
+
+It would also support any UTF-8 character. For example this array of names
+```json
+["染色体-1","染色体-2","染色体-3"]
+```
+
+Would create the following serialisation:
+
+```python
+b'["\xe6\x9f\x93\xe8\x89\xb2\xe4\xbd\x93-1","\xe6\x9f\x93\xe8\x89\xb2\xe4\xbd\x93-2","\xe6\x9f\x93\xe8\x89\xb2\xe4\xbd\x93-3"]'
+```
+
+#### 2. Digest of the canonical representation
+
+The canonical string representation is then digested. Assuming the use of GA4GH (sha512 trim to 24) digest, the following array of length
+
+```python
+b'[248956422,242193529,198295559]'
+```
+
+would be converted to 
+
+```json
+"5K4odB173rjao1Cnbk5BnvLt9V7aPAa2"
+```
+
+#### 3. Creation of an object composed of the array names and the digested arrays
+An object is created with the array name as properties and the digest as value.
```

Comment:
Do you want to include the decision to not add prefixes here, or is this a separate ADR?

Reply:
I decided to write another ADR for that -- what do you think?  https://github.com/ga4gh/seqcol-spec/pull/42

Reply:
I think splitting into a separate ADR is correct. That seemed to address the omission of `ga4gh:` but not the omission of `DT.` where `DT` is a condensed data type string. The data type though would be constant across all array digests so this might not be an issue and can be added once if the ID exits the system